### PR TITLE
Improve afl_fuzz wrapper

### PR DESCRIPTION
**Pull request by gnbon**
The current `ql_afl_fuzz` wrapper in Qiling supports setting exit addresses using the `exits` parameter. It sets the exit addresses at the start and calls the `ql_afl_fuzz_custom` wrapper with a dummy fuzz callback.

Although both `ql_afl_fuzz` and `ql_afl_fuzz_custom` wrappers have almost the same API, it is impossible to set exit addresses if `ql_afl_fuzz_custom` is called directly. 

To address this issue, I passed the exits parameter seamlessly to `ql_afl_fuzz_custom` for both wrappers to support setting exit addresses.

### Which kind of PR do you create?

- [x] This PR only contains minor fixes.
- [ ] This PR contains major feature update.
- [ ] This PR introduces a new function/api for Qiling Framework.

### Coding convention?

- [x] The new code conforms to Qiling Framework naming convention.
- [ ] The imports are arranged properly.
- [ ] Essential comments are added.
- [ ] The reference of the new code is pointed out.

### Extra tests?

- [x] No extra tests are needed for this PR.
- [ ] I have added enough tests for this PR.
- [ ] Tests will be added after some discussion and review.

### Changelog?

- [x] This PR doesn't need to update Changelog.
- [ ] Changelog will be updated after some proper review.
- [ ] Changelog has been updated in my PR.

### Target branch?

- [x] The target branch is dev branch.

### One last thing

- [x] I have read the [contribution guide](https://docs.qiling.io/en/latest/contribution/)

-----

### CODE REVIEW

**Comment by xwings** [comment]
@wtdcode can u take a look?

**Comment by wtdcode** [comment]
LGTM but I think you have to update samples.

**Comment by gnbon** [comment]
I have tested the sample scripts in the examples/fuzzing/ directory to ensure that they run smoothly without any modifications. 
For the stm32f429 using the `ql_afl_fuzz_custom` function, I have made the necessary modifications to comply with the updated API.

Additionally, I have checked the linux_x8664 and qnx_arm using the `ql_afl_fuzz` function and confirmed that they run seamlessly.

During the testing process, I encountered some issues that were unrelated to the `ql_afl_fuzz_*` wrappers in the dlink_dir815 and arm_tendaac15 samples. However, since these issues are outside the scope of this pull request, I will open an issue thread for them later.

**Comment by gnbon** [comment]
@wtdcode are these commits ready to be merged?

**Comment by wtdcode** [comment]
Yes, LGTM now. Let's wait for CI.

**Review by xwings — APPROVED** [review]

**Comment by xwings** [comment]
Thanks for the PR. Need to wait for one more approve before we are good to merge. @wtdcode

**Review by kabeor — APPROVED** [review]

**Pull request merged**
